README.md

```diff
@@ -1,4 +1,209 @@
+it is not possible to read the published version of an item variant that is published and has a [new version](https://kontent.ai/learn/docs/workflows-publishing/create-new-versions) at the same time.
+As a result, the export will only contain data of the latest versions and not the published ones.
+Published language variants that don't exist in any other workflow step are exported correctly.
+
+#### Language Variants Scheduled For Publishing
+
+With the current version of the Management API,
+it is not possible to get the time for which a variant is scheduled to be published.
+Therefore, the exported data only contain the information that the variant is scheduled to be published, but without the date.
+
+#### Performance
+
+The command leverages [the Management API](https://kontent.ai/learn/docs/apis/openapi/management-api-v2) to get the project data which might be slow for projects with large amounts of content items.
+
+## Import
+
+With the `import` command, you can import data into your Kontent.ai environment.
+The target environment needs to be empty, otherwise the command might fail.
+The command uses [the Management API](https://kontent.ai/learn/docs/apis/openapi/management-api-v2) to import the data.
+
+> [!TIP]
+> The command expects the data for import in a `.zip` file in the same [structure](#structure-of-the-exported-data) that is produced by the [export command](#export).
+>
+> If you want to import data from a different structure, you can use any available tool to convert it into the supported format.
+
+### Usage
+
+```bash
+npx @kontent-ai/data-ops import --fileName <file-to-import> --environmentId <target-environment-id> --apiKey <Management-API-key>
+```
+To see all supported parameters, run `npx @kontent-ai/data-ops import --help`.
+
+### Known Limitations
+
+#### Missing Entities
+
+Sitemap, roles and [asset type](https://kontent.ai/learn/docs/assets/asset-organization#a-set-up-the-asset-type) entities are not supported by the Management API and thus are not exported.
+
+#### Role limitations in Workflows
+
+Since it is not possible to import roles, the tool doesn't set role limitations when importing workflows.
+
+#### Web Spotlight
+
+It is not possible to enable [Web Spotlight](https://kontent.ai/learn/develop/hello-web-spotlight) through the Management API so it can't be "imported".
+As a result, it is also not possible to set root item for spaces, as this can only be done on environments with Web Spotlight enabled.
+
+#### New Versions of Published Variants
+
+Since the format doesn't support language variants with both a published version and a new one, we can't import it.
+See the [export command limitation](#new-versions-of-published-variants) for more details.
+
+#### Language Variants Scheduled For Publishing
+
+Because the format cannot support publishing time for variants scheduled to be published, the tool instead publishes all scheduled variants immediately during the import process.
```

Comment:
I think leaving the scheduled variants in Draft corresponds better to their original state, as scheduled variants are not published yet.